src/vrcosclib/Utility/OscUtility.Connection.cs

```diff
@@ -45,39 +46,34 @@ public static int ReceivePort
         }
     }
 
-    private static int _sendPort = 9000;
     public static int SendPort
     {
-        get => _sendPort;
+        get => _destination.Port;
         set
         {
-            if (value > 65535 || value < 0)
-            {
-                throw new ArgumentOutOfRangeException(nameof(value));
-            }
-            if (_sendPort == value)
-            {
-                return;
-            }
-
-            _sendPort = value;
-            if (_client != null)
-            {
-                _client.Dispose();
-                _client = null;
-            }
+            //Do we need this setter? Doesn't look so good...
```

Comment:
I will refine this comment later. 
If we remove the setter it can be a breaking change (if so I can update the tests as well). but could be OK since the project has just started... Please comment your thinking.